frontend/app_flowy/lib/workspace/presentation/home/home_layout.dart

```diff
@@ -36,10 +37,15 @@ class HomeLayout {
     if (forceCollapse) {
       showMenu = false;
     } else {
-      showMenu = context.widthPx > PageBreaks.tabletPortrait;
+      showMenu = true;
+      menuIsDrawer = context.widthPx <= PageBreaks.tabletPortrait;
     }
 
-    homePageLOffset = showMenu ? menuWidth : 0.0;
+    homePageLOffset = showMenu
```

Comment:
This statement is too complicated.

It is recommended to modify to `homePageLOffset = (showMenu && !menuIsDrawer) ? menuWidth : 0.0;`

Reply:
Thanks for the feedback, I have fixed it :+1: